KarhooUISDK/Translations/Base.lproj/Localizable.strings

```diff
@@ -640,6 +640,12 @@
 
 "Text.Quotes.Results" = "%1$@ results";
 
+"Text.Quotes.Filter" = "Filter";
+
+"Text.Quotes.FilterPageResults" = "View all %1$@ results";
```

Comment:
It would be nice if we will use localized strings with plurals (https://developer.apple.com/documentation/xcode/localizing-strings-that-contain-plurals) or at least separate label for no results state. Wih current implementation user will see: "View all 0 results".  This solution will require more work during translations, of course, but will be definitely more pro :)

Reply:
We can sort it out and implement within `quote list filter finish touch` task.